wherehows-web/app/routes/application.js

```diff
@@ -31,6 +32,19 @@ export default Route.extend(ApplicationRouteMixin, {
   },
 
   /**
+   * __It reloads the Ember.js application__ by redirecting the browser
+   * to the application's root URL so that all in-memory data (such as Ember
+   * Data stores etc.) gets cleared.
+   * @override ApplicationRouteMixin.sessionInvalidated
+   * @link https://github.com/simplabs/ember-simple-auth/issues/1048
+   */
+  sessionInvalidated() {
+    if (!testing) {
+      window.location.replace('/');
```

Comment:
@justinpark if you transition to you will get in an infinite loop, and the browser will still retain all the cached data from the previous session. replacing the route ensures the browser reloads the page, so if a user is logged out the browser will invalidate it's cache
https://github.com/simplabs/ember-simple-auth/blob/master/addon/mixins/application-route-mixin.js#L135